### PR TITLE
[No Ticket] Provide support language docs in main readme

### PR DESCRIPTION
**Pull request by meghfossa**
# Overview

This PR, 
- moves supported language listing to main readme (under details section)

Pretty Link: https://github.com/fossas/fossa-cli/tree/fix/include-prog-docs-in-readme

This is so, strategy docs are easily discoverable. Currently, it is hard to discover language specific docs, unless you use search function, or click 3-4 different links.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -140,6 +140,63 @@ Now that your analysis is complete, there are a couple things you might want to
 
 For most users, the FOSSA CLI will work out-of-the-box without any configuration. Just get an API key, run `fossa analyze`, and view your results in the FOSSA web application.
 
+<details>
+  <summary>
+    <b>Supported Languages</b>
+  </summary>
+
```
Comment (meghfossa) [inline README.md:147]: Text is from ./docs/references/strategy/README.md

Comment (jagonalez) [reply README.md:147]: Would having a link to the docs be enough?

If we have the same docs in two places - there's a high likely hood one will eventually become stale.

Comment (meghfossa) [reply README.md]: ✔️ with https://github.com/fossas/fossa-cli/pull/1324/commits/381dcfba4b1f1eff1a5e00b1f0d5b46b3f048a1e

**Review by jagonalez — APPROVED** [review]
LGTM

**Pull request merged**